lib/jsdom/browser/default-stylesheet.js

```diff
@@ -128,14 +123,6 @@ q {
     display: inline
 }
 
-q:before {
-    content: open-quote;
```

Comment:
nwmatcher [does not support `:before` or `:after`](https://github.com/dperini/nwmatcher/wiki/CSS-supported-selectors#content-pseudo-element-selectors-not-supported)